tests/SketchUp Ruby API/TC_Geom_Point2d.rb

```diff
@@ -2,6 +2,7 @@
 # License:: The MIT License (MIT)
 # Original Author:: Adam Karkkainen
 
+if Geom.const_defined? :Point2d
```

Comment:
What are these conditionals for?

Reply:
Oh, for running on older SketchUp versions?

Reply:
> Oh, for running on older SketchUp versions?

Yes.  I wanted to see if it worked with Ruby 2.2, so I tried SU 2017.

The '2d' objects were new with SU 2018, JFYI, doc's refer to Layout (not SU) in their minimum version item.

Reply:
Yea, the 2d classes was introduced because of LayOut. SketchUp never really used them. I think we might have introduced some usage of them later on though.